tests/sentry/workflow_engine/handlers/detector/test_base.py

```diff
@@ -165,7 +168,49 @@ def build_handler(
             detector = self.create_detector_and_conditions(detector_type)
         return MockDetectorStateHandler(detector)
 
-    def assert_updates(self, handler, group_key, dedupe_value, counter_updates, active, priority):
+    def assert_updates(
+        self,
+        handler: StatefulGroupingDetectorHandler,
+        group_key: DetectorGroupKey | None,
+        dedupe_value: int | None,
+        counter_updates: dict[str, Any] | None,
+        is_triggered: bool | None,
+        priority: DetectorPriorityLevel | None,
+    ):
+        """
+        Use this method when testing state updates that have been executed by evaluate
+        """
+        saved_state = handler.get_state_data([group_key])
+        state_data = saved_state.get(group_key)
+
+        if not state_data:
+            raise AssertionError(f"No state data found for group key: {group_key}")
+
+        if dedupe_value is not None:
+            assert state_data.dedupe_value == dedupe_value
+
+        if counter_updates is not None:
+            assert state_data.counter_updates == counter_updates
+
+        if is_triggered is not None:
+            assert state_data.is_triggered == is_triggered
+
+        if priority is not None:
+            assert state_data.status == priority
+
+    def assert_pending_updates(
```

Comment:
This looks unused, is it planned for use later on, or should we remove?

Reply:
this is the original implementation of `assert_updates` - i wasn't sure if we'd need it or not, seemed useful if we want to test state stuff in the future.